app/services/menu/loader.rb

```diff
@@ -13,11 +13,11 @@ def self.load
 
       Manager.map :user_menu do |menu|
         menu.item :my_account,
-                  :caption => N_('My account'),
+                  :caption => N_('My Account'),
```

Comment:
I wonder if it wont be better to auto correct the caption automatically, e.g. by using something like http://apidock.com//rails/ActiveSupport/Inflector/titleize